ui/raidboss/data/triggers/byakko-ex.js

```diff
@@ -6,36 +6,55 @@
     {
       id: 'ByaEx Heavenly Strike',
       regex: / 14:27DA:Byakko starts using Heavenly Strike on (\y{Name})/,
+      regexDe: / 14:27DA:Byakko starts using Himmelszorn on (\y{Name})/,
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Tank Buster on YOU';
+        return {
```

Comment:
Would you mind indenting these returns and adding curly braces around the conditionals with multi-line bodies, like this:

```
        if (matches[1] == data.me) {
          return {
            en: 'Tank Buster on YOU',
            de: 'Tankbuster auf DIR',
          };
        }
```

...both on this line, but elsewhere in these diffs too.

I'm looking into something like https://github.com/beautify-web/js-beautify to do this automatically but I haven't set that up yet (or committed to a particular pretty printer).

Reply:
Sure, I can do that ^^